mod/fabric/src/main/resources/fabric.mod.json

```diff
@@ -1,18 +1,20 @@
 {
 	"schemaVersion": 1,
-	"id": "map-sync",
-	"version": "${version}",
-	"name": "Map Sync",
-	"description": "Sync map data with others instantly.",
+	"id": "mapsync",
+	"version": "${mod_version}",
+	"name": "${mod_display_name}",
+	"description": "${mod_description}",
 	"authors": [
-		"Gjum"
+		"Gjum",
+		"Protonull"
```

Comment:
Would you like to add the other contributors as well while you're at it?

Reply:
Who should I add and do you have a preferred order?

Reply:
Gjum
Protonull
okx
Huskydog9988
specificlanguage
SirAlador
klaribot